packages/server/database/migrations/20221012142324-fixDakiMigration.ts

```diff
@@ -0,0 +1,23 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function (r: R) {
+  const now = new Date()
+  const missingKeepPrompt = {
+    createdAt: now,
+    description: 'What should we continue doing?',
+    groupColor: '#D9D916',
+    id: 'keepPrompt',
+    isActive: true,
+    question: 'Keep',
+    sortOrder: 2,
+    teamId: 'aGhostTeam',
+    templateId: 'dropAddKeepImproveDAKITemplate',
+    title: 'Keep',
+    updatedAt: now
+  }
+  await r.db('actionDevelopment').table('ReflectPrompt').insert(missingKeepPrompt).run()
```

Comment:
-1 this is the wrong database. Usually we don't put the database here at all because it is configured somewhere.